src/kvilib/system/KviEnvironment.cpp

```diff
@@ -33,14 +33,14 @@ namespace KviEnvironment
 
 #if !defined(COMPILE_ON_WINDOWS) && !defined(COMPILE_ON_MINGW)
 
-#ifdef HAVE_SETENV
-	bool setVariable(const char * name, const char * value)
+	bool setVariable(QString szName, QString szValue)
```

Comment:
The `QString`s passed should be passed as `const QString&` to avoid an unnecessary copy.